app/dashboard/templates/dashboard/index.html

```diff
@@ -34,9 +34,37 @@
       {% include 'shared/top_nav.html' with class='d-md-flex' %}
       {% include 'shared/nav.html' %}
     </div>
-    {% include 'dashboard/ftux.html' %}
+    {% if hackathon %}
+      <div class="row pt-5 pb-5 text-center" style="
+        background: rgb(25, 0, 63);
+        background-image: url({% static '/v2/images/header-bg.png' %});
+        background-size: cover;
+      ">
+        <div class="col g-font-muli">
+          <span class="d-block font-title-xl text-white">{{ hackathon.name }}</span>
+          {% if hackathon.logo_svg %}
+            <img class="d-block mx-auto my-4" max-width="364" src="{{ hackathon.logo_svg.url }}"/>
+          {% elif hackathon.logo %}
+            <img class="d-block mx-auto my-4" max-width="364" src="{{ hackathon.logo.url }}"/>
+          {% endif %}
+          <span class="d-block font-title-xl text-white font-weight-300" style="white-space: nowrap;">
+            {{ hackathon.start_date|date:"M j, Y" }} - {{ hackathon.end_date|date:"M j, Y" }}
+          </span>
+        </div>
+      </div>
+    {% else %}
+      {% include 'dashboard/ftux.html' %}
+    {% endif %}
     <div id="dashboard-content" class="container-fluid no-gutter">
       <div class="row">
+        {% if hackathon %}
+        <div class="col-12 col-lg-12 col-xl-12 body">
+            <div class="container p-2" id="bounties">
+              <div class="col-12 col-lg-12 pl-2 pb-1 title-row">
+                <span class="font-title ml-1 text-highlight-gc-blue">{% trans 'Bounties' %}</span>
+              </div>
+              <div style="clear: right;"></div>
+        {% else %}
         <div class="col-12 col-lg-3 col-xl-2 invisible" id="sidebar_container" style="background: #F2F6F9;">
```

Comment:
indent this bit by 1 cause it's wrapped within the `else` block

Reply:
to much spaghetti here